run.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -x
+set -e
+[ -n "${CLEAN_CONFIG:-}" ] && rsync -a --delete /app/config-orig/ /app/config/
+[ -f "/app/config/global.php" ] || rsync -a --delete /app/config-orig/ /app/config/
+chown -R www-data:www-data /app/config
+/etc/init.d/apache2 start
```

Comment:
Yes in upstream run.sh script, they load environments ... like /etc/init.d/apache2 do that, and they start apache in foreground mode. I prefer used Ubuntu / Debian apache maintainer script and follow evolution made by this team. 
And yes, we can start run.sh given by tutum team after initialization ... but tutum team is more little and less reactive than Debian and Ubuntu community :-D